Documentation/ceph-quickstart.md

```diff
@@ -40,7 +38,7 @@ The first step is to deploy the Rook system components, which include the Rook a
 cd cluster/examples/kubernetes/ceph
 kubectl create -f operator.yaml
 
-# verify the rook-ceph-operator and rook-ceph-agent pods are in the `Running` state before proceeding
+# verify the rook-ceph-operator, rook-ceph-agent, and rook-ceph-discover pods are in the `Running` state before proceeding
```

Comment:
the discover pod doesn't actually have ceph in its name, e.g.`rook-discover-gsgzw`.